test/test_nn.py

```diff
@@ -9340,14 +9341,32 @@ def test_conv_transposed_large(self, device):
         self.assertEqual(maxdiff2, 0)
         self.assertEqual(maxdiff3, 0)
 
+    @skipIfRocm
     @unittest.skipIf(not TEST_LARGE_TENSOR, "not enough memory to run test")
     def test_conv_large(self, device):
         dtype = torch.half if self.device_type == 'cuda' else torch.float
-        conv1 = nn.Conv2d(2, 2, 8, 8).to(device).to(dtype)
+        conv = nn.Conv2d(2, 2, 8, 8, bias=False).to(device).to(dtype)
         input_large = torch.randn(4096, 2, 512, 512, dtype=dtype, device=device)
-        ret = conv1(input_large)
-        self.assertEqual(ret[:2048], conv1(input_large[:2048]))
-        self.assertEqual(ret[2048:], conv1(input_large[2048:]))
+        # forward
+        ret = conv(input_large)
+        self.assertEqual(ret[:2048], conv(input_large[:2048]))
+        self.assertEqual(ret[2048:], conv(input_large[2048:]))
+
+        # backward
+        conv.zero_grad()
+        ret.view(4096, -1).max(dim=1).values.sum().backward()
```

Comment:
what's wrong with a simple sum() here?

Reply:
a simple `sum()` means all the outputs have contributions, therefore in the backward, we are summing lots of values, the rounding errors could then become as large as `1e-5`, larger than the threshold of `assertEqual` (`1e-6` I guess). Having a `.max(dim=1)` makes it sparse to reduce the rounding errors.

Reply:
Cool, can you then please leave a comment about this?

Reply:
Sure

Reply:
Comment added